Makefile

```diff
@@ -3,7 +3,9 @@ BINARY			= utask
 MAIN_LOCATION	= ./cmd
 
 TEST_LOCATION	= ./...
-TEST_CMD		= go test -count=1 -v -cover -p 1 ${TEST_LOCATION}
+# timeout for go test is here to prevent test running infinitely if one runResolution lead to a step that never recovers (missing push on a stepChan)
```

Comment:
```suggestion
# timeout for go test is here to prevent tests running infinitely if one runResolution leads to a step that never recovers (e.g. missing push on a stepChan)
```